deps/src/main/java/com/microsoft/azure/sdk/iot/deps/serializer/ParserUtility.java

```diff
@@ -456,7 +456,7 @@ else if (jsonElement.isJsonArray())
         }
     }
 
-    public static Object getJsonPrimitiveValue(JsonPrimitive jsonPrimitive)
+    private static Object getJsonPrimitiveValue(JsonPrimitive jsonPrimitive)
```

Comment:
I'm so-so on changing these two methods. Since it is a utility it might make sense to keep these as public. We're not really hiding any need-to-be-changed implementations here.

Reply:
This utility class itself is buried far enough away from our public APIs that I'd be surprised if anyone was using it. We can always make it public again later if someone asks for it